### PR TITLE
refactored the follower.py and created post class

### PR DESCRIPTION
**Pull request by alexbara2000**
I created the class post that will store the info for each post like the user, image, likes, comments and ID. 
refactored the follower class to be independent of the database we use.

#37 #39

### CODE REVIEW

File: posts.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        self.user = user
+        self.imgpath = imgpath
+        self.time = time.time()
+        self.comments = []
+        postsList = getListFromCSV('data/posts.csv')
+        self.postID = len(postsList)-1
+
+
+
+def addpost(user, imgpath):
```
Comment (LiamBurchellReyes) [inline posts.py]: Capitalization: try "addPost" instead?

**Review by LiamBurchellReyes — CHANGES_REQUESTED** [review]
Small change: Just fix the capitalization on the addpost function to lowerCamelCase. Excellent otherwise. Good refactor.

**Review by LiamBurchellReyes — APPROVED** [review]
Excellent.

**Review by MatteoGisondi — APPROVED** [review]
Many additions changed in synchronization branch.

**Pull request closed without merging**